components/Tooltip/TooltipStyles.tsx

```diff
@@ -32,7 +32,8 @@ export const Tooltip = styled.div<ToolTipStyleProps>`
 			font-family: ${(props) => props.theme.fonts.mono};
 			font-style: normal;
 			font-weight: 400;
-			line-height: 12px;
+			line-height: 8px;
```

Comment:
Can we increase the `line-height` to 125% and remove the `newline` below?

Reply:
125% did the trick